documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -135,15 +134,18 @@ If your cluster already uses `KafkaNodePool` custom resources, and you wish to d
 [id='ref-operator-unidirectional-topic-operator-feature-gate-{context}']
 == (Preview) UnidirectionalTopicOperator feature gate
 
-The `UnidirectionalTopicOperator` feature gate has a default state of _disabled_.
+The `UnidirectionalTopicOperator` feature gate has a default state of _enabled_.
 
 The `UnidirectionalTopicOperator` feature gate introduces a unidirectional topic management mode for creating Kafka topics using the `KafkaTopic` resource.
 Unidirectional mode is compatible with using KRaft for cluster management.
 With unidirectional mode, you create Kafka topics using the `KafkaTopic` resource, which are then managed by the Topic Operator.
 Any configuration changes to a topic outside the `KafkaTopic` resource are reverted.
 For more information on topic management, see xref:ref-operator-topic-str[].
 
-.Enabling the UnidirectionalTopicOperator feature gate
+.Disabling the `UnidirectionalTopicOperator` feature gate
 
-To enable the `UnidirectionalTopicOperator` feature gate, specify `+UnidirectionalTopicOperator` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
-For the `KafkaTopic` custom resource to use this feature, the `strimzi.io/managed` annotation is set to `true` by default. 
\ No newline at end of file
+To disable the `UnidirectionalTopicOperator` feature gate, specify `-UnidirectionalTopicOperator` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
+
+.Downgrading from UnidirectionalTopicOperator
+
+If you want to downgrade to the previous Topic Operator implementation, you can simply disable the feature gate and then remove all the `KafkaTopic` finalizers.
```

Comment:
Should you cover also the downgrade  / upgrade of the operator?

Reply:
There is another PR from Paul that is also dealing with that.